docs/reference/faq.mdx

```diff
@@ -102,6 +102,18 @@ wasmCloud is compatible with, but not dependent on, Kubernetes. wasmCloud hosts
 
 wasmCloud is used in production by several companies already. See the [deployment guide](/docs/deployment/) for topics related to running wasmCloud in production.
 
+### How does wasmCloud handle "noisy neighbor" issues?
+
+This is a common question for anyone running a large distributed system! In terms of wasmCloud, there are two major ways this problem can occur: CPU/memory resource contention (normal for most systems) and [provider](/docs/reference/glossary#capability-providers) bottlenecks. Both of these are covered below:
+
+#### CPU/Memory Resource Contention
+
+The most common type of noisy neighbor problem is resource contention over things like CPU and memory. Because Wasm is an emerging technology there isn't a 100% standard way to handle this yet. However, we have plans to add resource scheduling to [Wadm](https://github.com/wasmcloud/wadm), our scheduling and orchestration tool. This will recognize requests and limits for actors and providers, and will ensure that resources are available for use, returning an error if things would be overprovisioned. Right now, you'll still have to manage enforcement of these limits at the OS or container level (which is currently the status quo for most things). However, in the future, because Wasm is extremely deterministic, we should be able to use a combination of the metrics a wasmCloud host exposes and knowledge of what steps a module will take to develop a much more flexible, dynamic, and automated way of handling limits and requests of resources.
```

Comment:
Consider mentioning that while we don't yet have resource limits in terms of CPU/memory, we _do_ have limits in terms of concurrency, which can be a useful proxy. It's hard to be a noisy neighbor if only 10 actor instances can be active at the same time

Reply:
Good point, I've added it